ray-operator/apis/ray/v1alpha1/rayservice_types.go

```diff
@@ -0,0 +1,48 @@
+package v1alpha1
```

Comment:
I see you rename the fold and put everything under same `ray/v1alpha1`.  That means

1. All crds should have exact same group version
2. If we like to bump the version, we may bump all crd together? Or just create partial resources in v1alpha2 or v1beta1?

Reply:
@DmitriGekhtman I think we have a discussion before. We want to put them in the same group. But I am open to discuss that more.
From [kubebuilder](https://book.kubebuilder.io/multiversion-tutorial/tutorial.html), we should be able to maintain CRDs with different upgrading version. The crd types have versioning, but the controllers are the same. What required should be the conversion between versions.